webview-ui/src/components/chat/ChatView.tsx

```diff
@@ -1802,7 +1802,7 @@ const ChatViewComponent: React.ForwardRefRenderFunction<ChatViewRef, ChatViewPro
 												appearance="primary"
 												disabled={!enableButtons}
 												className={secondaryButtonText ? "flex-1 mr-[6px]" : "flex-[2] mr-0"}
-												onClick={() => handlePrimaryButtonClick(inputValue, selectedImages)}>
```

Comment:
I think there might have been some intentional change here around including the message when approving or rejecting the ask. Does this change that behavior?